src/tools/cephfs/cephfs-shell

```diff
@@ -855,7 +855,7 @@ sub-directories, files')
             if index == 0:
                 self.poutput('{:25s}\t{:5s}\t{:15s}{:10s}{}'.format(
                     "1K-blocks", "Used", "Available", "Use%", "Stored on"))
-            if not is_dir_exists(i.d_name):
+            if is_dir_exists(i.d_name):
```

Comment:
This looks like a separate bug. Please create a new PR for this change.

Reply:
Done.